src/main/java/step3/racingcar/Car.java

```diff
@@ -0,0 +1,27 @@
+package step3.racingcar;
+
+public class Car {
+
+    private static final int DEFAULT_POSITION = 0;
+    private int position;
+    private final MovingStrategy movingStrategy;
+
+    public Car(MovingStrategy movingStrategy) {
+        this.position = DEFAULT_POSITION;
+        this.movingStrategy = movingStrategy;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void moveCar() {
+        boolean isMovable = this.movingStrategy.isMovable();
+        moveCarForward(isMovable);
+    }
+
+    public void moveCarForward(boolean isMovable) {
+        if (isMovable == true)
+            this.position++;
+    }
```

Comment:
개인적인 의견인데요, 이정도는 분리하지 않아도 괜찮을 것 같아요

```suggestion
    public void moveCarForward(boolean isMovable) {
        if (this.movingStrategy.isMovable() == true)
            this.position++;
    }
```